<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramaWindow</class>
 <widget class="QWidget" name="ProgramaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="cliente_c_box"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="cliente_label">
       <property name="text">
        <string>Cliente</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="add_treino">
       <property name="text">
        <string>Add Treino</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0" colspan="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="3" colspan="4">
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>528</width>
          <height>415</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="programa_layout"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="salvar_button">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
